frontend/src/pages/home/HomePages.tsx

```diff
@@ -2,18 +2,19 @@ import HomePage from './HomePage.tsx';
 import Home from "./Home.tsx";
 import {useLoaderData} from "react-router-dom";
 import {ProjectDeadline} from "../project/projectDeadline/ProjectDeadline.tsx";
+import {Me} from "../../types/me.ts"
 
 /**
  * Gives the requested home page based on the login status
  * @returns - The home page component 
  */
 export default function HomePages() {
   const loader = useLoaderData() as {
-    projects: ProjectDeadline[],
-    me: string
-  }
-  const me = loader.me
-  if (me === 'UNKNOWN') {
+    projects: ProjectDeadline[];
+    me: Me;
+  };
+  const me = loader.me.role;
+  if (me === "UNKNOWN") {
```

Comment:
Same here, make a function that checks if someone is logged in, the meaning of roles can change which can cause this to break